Makefile

```diff
@@ -338,7 +338,7 @@ apidiff: $(GO_APIDIFF) ## Check for API differences
 ALL_VERIFY_CHECKS = boilerplate modules gen conversions
 
 .PHONY: verify
-verify: $(addprefix verify-,$(ALL_VERIFY_CHECKS)) ## Run all verify-* targets
+verify: $(addprefix verify-,$(ALL_VERIFY_CHECKS)) lint-markdown lint-shell ## Run all verify-* targets
```

Comment:
Adding those two linters so we can get rid of the separate jobs to simplify our job landscape

Reply:
Requires adding docker to the prowjob?!

Reply:
Yup PR will be opened in a bit

Reply:
@chrischdi https://github.com/kubernetes/test-infra/pull/30246